tests/db/migration_tests/test_v202209160_to_v202212010.py

```diff
@@ -0,0 +1,65 @@
+"""
+    Copyright 2022 Inmanta
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Contact: code@inmanta.com
+"""
+import os
+from typing import AsyncIterator, Awaitable, Callable, List
+
+import pytest
+from asyncpg import Connection
+
+from db.common import PGRestore
+from inmanta.server.bootloader import InmantaBootloader
+
+
+@pytest.fixture
+async def migrate_v202209160_to_v202212010(
+    hard_clean_db, hard_clean_db_post, postgresql_client: Connection, server_config
+) -> AsyncIterator[Callable[[], Awaitable[None]]]:
+    """
+    Returns a callable that performs a v202209160 database restore and migrates to v202212010.
+    """
+    # Get old tables
+    with open(os.path.join(os.path.dirname(__file__), "dumps/v202209160.sql"), "r") as fh:
```

Comment:
Why start from `v202209160`  there is a newer version already `v202211230`?

Reply:
I don't see this dump?